.github/workflows/e2e.yaml

```diff
@@ -93,35 +93,12 @@ jobs:
         # output_dir
         mkdir -p _artifacts
         # create cluster
-        cat <<EOF | /usr/local/bin/kind create cluster \
+        sed -i  's/IP_FAMILY/'${IP_FAMILY}'/g' examples/kind/kind.config
+        sed -i  's#EGREE_SELECTOR_CONFIGURATION_PATH#./examples/kind/egress_selector_configuration.yaml#g' examples/kind/kind.config
```

Comment:
I can see what you mean, but this use case is in CI and unlikely to pick it up  by git. Can you explain a little more? Thanks. :)

I can revert this part If you think it would be more appropriate to use EOF to complete the processing of the configuration files in CI .

Reply:
Let me revert it to eliminate noise caused by environment variables.